<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdtGraph</class>
 <widget class="QDialog" name="EdtGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑图</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>顶点数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledt_vnum">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="edt_graphinfo">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_dlgok">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_dlgrand">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>随机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_dlgcancel">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
